### PR TITLE
refactor: common asset processor

### PR DESCRIPTION
**Pull request by Davidmattei**
Moved image/asset processor logic from core to common bundle.
The core and the skeleton will use the common processor in their controllers.

Other pull requests:
- https://github.com/ems-project/website-skeleton/pull/10
- https://github.com/ems-project/elasticms/pull/16
- https://github.com/ems-project/EMSCoreBundle/pull/94
- https://github.com/ems-project/EMSCommonBundle/pull/7

### CODE REVIEW

File: Helper/Routing/Router.php [hunk 1]
```diff
@@ -50,9 +53,23 @@ public function buildRouteCollection(): void
         $this->addLanguageSelectionRoute($collection);
         $this->addSearchResultRoute($collection);
 
+        if ($this->assetConfigType) {
```
Comment (Davidmattei) [inline Helper/Routing/Router.php]: Improve the assetConfig is also in the clientRequest Options

File: Resources/config/routing.xml [hunk 2]
```diff
@@ -3,10 +3,11 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services
         http://symfony.com/schema/dic/services/services-1.0.xsd">
-
+    <parameters>
+        <parameter key="env(EMSCH_ASSET_CONFIG_TYPE)"/>
```
Comment (Davidmattei) [inline Resources/config/routing.xml]: Not good

File: Helper/Asset/ProcessHelper.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        $this->processor = $processor;
+        $this->clientRequest = $clientRequestManager->getDefault();
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function process(Request $request, string $processor, string $assetHash, string $configHash = null): Response
+    {
+        if ($configHash) {
+            //already generated by twig function!
```
Comment (dameert) [inline Helper/Asset/ProcessHelper.php]: dead code

Comment (dameert) [reply Helper/Asset/ProcessHelper.php]: if that is covered, this comment is dead code..

Comment (Davidmattei) [reply Helper/Asset/ProcessHelper.php]: True will be removed

**Pull request merged**